src/app/public/modules/phone-field/phone-field.component.html

```diff
@@ -0,0 +1,45 @@
+<div
+  class="sky-form-group sky-input-group"
```

Comment:
Should the outer wrapper of this component have a `sky-phone-field` class for easy DOM retrieval? I noticed we have this pattern on a few components and its a nice way to find stuff.

Reply:
Done